audit-service/src/main/java/io/vertx/workshop/audit/impl/AuditVerticle.java

```diff
@@ -87,10 +87,7 @@ private void retrieveOperations(RoutingContext context) {
 
   private Future<MessageConsumer<JsonObject>> retrieveThePortfolioMessageSource() {
     Future<MessageConsumer<JsonObject>> future = Future.future();
-    MessageSource.getConsumer(discovery,
-        new JsonObject().put("name", "portfolio-events"),
-        future.completer()
-    );
+    MessageSource.getConsumer(discovery, new JsonObject().put("name", "portfolio-events"), future);
```

Comment:
so actually I have only rx-ified the solution and not yet copied to the audit verticle, will